efficientdet/main.py

```diff
@@ -284,13 +297,21 @@ def _eval(steps):
         input_rand_hflip=False,
         is_training_bn=False,
     )
-    eval_estimator = tf.estimator.tpu.TPUEstimator(
-        model_fn=model_fn_instance,
-        use_tpu=use_tpu,
-        train_batch_size=FLAGS.train_batch_size,
-        eval_batch_size=FLAGS.eval_batch_size,
-        config=run_config,
-        params=eval_params)
+    if FLAGS.strategy == 'tpu':
+      eval_estimator = tf.estimator.tpu.TPUEstimator(
+          model_fn=model_fn_instance,
+          use_tpu=use_tpu,
+          train_batch_size=FLAGS.train_batch_size,
+          eval_batch_size=FLAGS.eval_batch_size,
+          config=run_config,
+          params=eval_params)
+    else:
+      eval_params['batch_size'] = \
```

Comment:
Prefer something like

eval_params['batch_size'] =  (
    FLAGS.eval_batch_size // getattr(strategy, 'num_replicas_in_sync', 1))

Internally, we almost never use "\" (It is a warning)